### ISSUE TITLE
Kodi 18 autoplay paused video after PC wakeup

### ISSUE THREAD

**Issue by kajman-cz** [xbmc/xbmc#15737]
### Describe the bug
Here is a clear and concise description of what the problem is:

Kodi 18 autoplay paused video after PC wakeup.

## Expected Behavior
Here is a clear and concise description of what was expected to happen:

Keep video paused (like Kodi 17).

## Actual Behavior

Video automatically resumes playback.

## Possible Fix

Add a switch to settings what to do with the paused video after waking up system.

### To Reproduce

1. Play video
2. Pause video
3. Switch PC to sleep mode
4. Wake up PC

### Debuglog

The debuglog can be found here:

### Screenshots 
Here are some links or screenshots to help explain the problem:

## Additional context or screenshots (if appropriate)
Here is some additional context or explanation that might help:

### Your Environment
Used Operating system:

 - [ ] Android
 - [ ] iOS
 - [ ] Linux
 - [ ] OSX
 - [ ] Raspberry-Pi
 - [x ] Windows
 - [ ] Windows UWP

 - Operating system version/name: Windows 7
 - Kodi version: 18.0, 18.1

*note: Once the issue is made we require you to update it with new information or Kodi versions should that be required.
Team Kodi will consider your problem report however, we will not make any promises the problem will be solved.*

**Comment by taintedkernel** [comment]
Also experiencing this, it's a minor but very annoying bug.  Has there been any investigation into it?

**Comment by jddcef** [comment]
I actually get the same thing on the Android version...

**Comment by ksooo** [comment]
For one it is a bug, for others a feature. If I remember correctly, what you are calling a bug was requested by others as a feature.

**Comment by ksooo** [comment]
One more question: you pause the video, then suspend and want to keep it paused on resume? If so, we might actually gave a bug, because automatically unpausing seems wrong.

What was requested by people as a feature is to automatically continuepayback on system resume if the video/song was playing when system was suspended.

**Comment by JiriSko** [comment]
Yeah, I want to have video paused when it was paused before suspending. And it's not happening right know.

**Comment by micmac1** [comment]
Also after resume the current playlist doesn't seem to work. Kodi finishes the current video and then just stops. The remaining items on the playlist don't get played. I'm on 19.1 currently. Even hitting the remote's key to skip to the next file doesn't work. If you open the playlist it actually shows all the files, though.

**Comment by fuzzard** [comment]
As crazy as it may sound, that's not actually a "bug". For most (all?) Platforms that support sleep/suspend, Kodi doesn't save/restore playlist items. All we do is store the currently playing item, and that's the item that restarts.

**Comment by micmac1** [comment]
Thanks for your comment! To me it all seems a bit weird. My expectation is that the state stays the same after resume. Anyway, just wanted to mention this. Thanks!

**Comment by mmortal03** [comment]
I'm seeing the same in Kodi 22.0-ALPHA1. Like ksooo said, "you pause the video, then suspend and want to keep it paused on resume? If so, we might actually gave a bug, because automatically unpausing seems wrong." 

Definitely a bug. I wonder what causes it? Could it just be that the most straightforward (but incomplete) programmatic way to continue playback of content that *isn't* paused at suspend has been to have Kodi initiate playback at resume no matter the context? 
Does Kodi even detect when the system is being suspended and, if the content is playing, initiate a pause? That would sort of make sense to then have to initiate play on resume, but I don't know what the code is actually doing. 
Basically, the generic logic should be: if the state of the content is paused at suspend, don't initiate play at resume. If the state of the content is playing at suspend, initiate play at resume.
But, arguably, some people might want it to *not* initiate play at resume *even* if it was playing at suspend. 
So, I could imagine there being a setting with at least two choices, like: 1.) Never start playback when resuming from suspend, 2.) Start playback when resuming from suspend if content was playing on suspend.
